README.md

```diff
@@ -49,10 +49,7 @@ To learn more about developing for Android, see [docs/android-development](docs/
 
 ### Building for Android with Docker
 
-A Docker image with all pre-requisites for Android builds is included.  To build:
-
-* Install dependencies with `./tools/build/build.sh npm ci`
```

Comment:
Don't you need to install the dependencies first?

Reply:
I don't get it. Where is the container getting the dependencies from?

Reply:
This is what I get when i run `./tools/build/build.sh npm ci` btw:

```
daniellacosse-macbookpro% ./tools/build/build.sh npm ci
npm WARN prepare removing existing node_modules/ before installation
The authenticity of host 'github.com (140.82.114.4)' can't be established.
RSA key fingerprint is SHA256:nThbg6kXUpJWGl7E1IGOCspRomTxdCARLviKw6E5SY8.
Are you sure you want to continue connecting (yes/no)? The authenticity of host 'github.com (140.82.114.4)' can't be established.
RSA key fingerprint is SHA256:nThbg6kXUpJWGl7E1IGOCspRomTxdCARLviKw6E5SY8.
Are you sure you want to continue connecting (yes/no)? The authenticity of host 'github.com (140.82.114.4)' can't be established.
RSA key fingerprint is SHA256:nThbg6kXUpJWGl7E1IGOCspRomTxdCARLviKw6E5SY8.
Are you sure you want to continue connecting (yes/no)? The authenticity of host 'github.com (140.82.114.4)' can't be established.
RSA key fingerprint is SHA256:nThbg6kXUpJWGl7E1IGOCspRomTxdCARLviKw6E5SY8.
Are you sure you want to continue connecting (yes/no)? 
```

Reply:
maybe `./tools/build/build.sh` pulls an image from quay, which has all the dependencies installed already? 🤷‍♂️ or it was mounting something idk

this has seemed to stop working for me randomly now, so i'm going to revert it anyway. it's getting hung up on the `webpack` call:

```
+ webpack --config=src/www/cordova.webpack.js
/Users/daniellacosse/outline-client/src/www/build_cordova_action.sh: /Users/daniellacosse/outline-client/node_modules/.bin/webpack: /usr/bin/env: bad interpreter: Permission denied
npm ERR! code ELIFECYCLE
npm ERR! errno 126
npm ERR! outline-client@ do: `bash ./scripts/do_action.sh "src/www/build_cordova"`
npm ERR! Exit status 126
npm ERR! 
npm ERR! Failed at the outline-client@ do script.
```

But probably related to the fact I can't run `./tools/build/build.sh npm ci`, not that i removed `npx`